test/e2e/tokens/erc20.test.mjs

```diff
@@ -400,7 +400,7 @@ describe('ERC20 tests', () => {
       }
     });
 
-    it('should restrict withdrawals', async () => {
+    it('should restrict withdrawals', async function () {
```

Comment:
Why this change?

Reply:
[Because `this` is bound to the wrong scope when you use arrow functions](https://stackoverflow.com/questions/51491553/mocha-using-this-skip-to-skip-tests-dynamically-doesnt-work)